src/style.css

```diff
@@ -1,3 +1,450 @@
+  font-weight: 300;
+  letter-spacing: 20%;
+}
+
+.save {
+  position: relative;
+  top: 5px;
+  margin-right: 5px;
+}
+
+.hr {
+  margin-top: 15px;
+  height: 1px;
+  width: 95%;
+  background-color: #909090;
+}
+
+.items {
+  display: flex;
+  gap: 20px;
+}
+
+.items a {
+  color: white;
+  border: 1px solid white;
+  text-decoration: none;
+  text-transform: capitalize;
+  padding: 5px 10px;
+  border-radius: 25px;
+}
+
+.items a:first-child {
+  background-color: white;
+  color: rgb(0, 0, 0);
+}
+
+/* items-video */
+
+.video {
+  display: flex;
+  gap: 10px;
+  margin-top: 10px;
+}
+
+.video img {
+  margin-top: 13px;
+  max-width: 45%;
+}
+
+.video p {
+  font-size: 12px;
+}
+
+p:nth-child(2) .video-articles p:nth-child(3) {
+  color: silver;
+}
+
+p:nth-child(3) .video-articles {
+  margin-top: 3px;
+}
+
+.new {
+  background-color: rgba(192, 192, 192, 0.247);
+  width: 25%;
+  text-align: center;
+}
+
+.new div {
+  margin-top: 22px;
+}
+
+.times1 {
+  display: flex;
+}
+
+.time-video {
+  text-align: center;
+  position: relative;
+  right: 85px;
+  top: 10px;
+  width: 24%;
+  height: 17px;
+  background-color: rgba(0, 0, 0, 0.815);
+}
+
+/* channel arrea */
+
+.channel {
+  display: flex;
+  justify-content: space-between;
+}
+
+.channel-sub {
+  display: flex;
+  margin-right: 45px;
+  margin-bottom: 5px;
+  align-items: center;
+}
+
+p .channel-sub {
+  font-size: 12px;
+  margin-right: 10px;
+  background-color: rgba(255, 255, 255, 0.233);
+  padding: 5px 8px 5px 8px;
+  text-transform: uppercase;
+}
+
+.channel-sub i {
+  margin-bottom: 2px;
+  font-size: 15px;
+  color: rgba(255, 255, 255, 0.61);
+}
+
+.channel-name {
+  display: flex;
+  align-items: center;
+}
+
+.channel-name img {
+  max-width: 15%;
+  border-radius: 50%;
+  margin-top: 10px;
+}
+
+.channel-name .names {
+  padding-left: 15px;
+  display: block;
+}
+
+.names span {
+  font-size: 13px;
+  color: rgba(255, 255, 255, 0.61);
+}
+
+p .titles {
+  color: rgba(255, 255, 255, 0.842);
+  font-size: 12px;
+  margin-left: 60px;
+}
+
+.show-more {
+  margin-left: 58px;
+}
+
+.show-more::after {
+  position: relative;
+  top: 15px;
+  display: block;
+  height: 1px;
+  background-color: #dfdbd7;
+  content: " ";
+  width: 730px;
+}
+
+p .show-more {
+  margin-top: 20px;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.comment {
+  position: relative;
+  top: 30px;
+  left: 55px;
+}
+
+.comment-num {
+  display: flex;
+  align-items: center;
+  gap: 35px;
+}
+
+p .comment-num {
+  font-size: 15px;
+  margin-bottom: 10px;
+}
+
+.comment-num span {
+  color: white;
+  margin-left: -20px;
+  font-size: 15px;
+}
+
+.comment-sort {
+  display: flex;
+  align-items: center;
+  gap: 15px;
+}
+
+.comment-sort img {
+  max-width: 5%;
+  border-radius: 50%;
+}
+
+.hr2 {
+  margin-top: 7px;
+  height: 1px;
+  width: 680px;
+  background-color: #909090;
+}
+
+.whole-comment {
+  margin-top: 70px;
+}
+
+.single-comment {
+  margin: 20px 0 0 55px;
+  display: flex;
```

Comment:
- [ ] Avoid code repetion, you have this `display: flex;` 15 times, please fix this, you can use a class and apply that class everywhere this property is needed